CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.6]
+
+**Added**
+
+- More `tfe.keras` functionality to convert `tf.keras` model into `tfe.keras`:
+   - `set_weights`
+   - `from_config`
+   - `model_from_config`
+   - `clone_model`
+- Example with conversion of `tf.keras` model into `tfe.keras`.
+- Better exception handling when `.so` files required for secure randomness custom op are missing.
```

Comment:
This sentence is a little confusing, perhaps:

`Improved handling for cases where the secure random operation is not available`

Reply:
sounds good